web/src/app/profile/approved-researcher/components/AgreementText.css

```diff
@@ -0,0 +1,19 @@
+.section {
+  padding-top: 1rem;
+
+  p {
+    padding-top: 0.7rem;
+  }
+
+  li {
+    padding-top: 0.2rem;
+  }
+
+  a {
+    color: #6900ff;
+  }
+
+  a:hover {
+    text-decoration: underline;
+  }
+}
```

Comment:
I'm pretty sure I'm in the minority when it comes to CSS nesting. I've been using the old-school approach for so long that I'm still not quite used to it. @acholyn and I had a chat about whether to use nesting or not. If both of you want to use it then I will take this project as my gateway in!

Reply:
It's my preference but if we do run into issues where it's not being interpreted correctly we should go old-school. It's been fine for all projects I've used it on since it was introduced in 2023 though so hopefully this will not go awry

Reply:
Yeah, I don't predict any compatibility issues as it is established now. It was simply a personal hesitancy 😅